### DISCUSSION TITLE
dont keep the prompt line ones that the command is executed

### DISCUSSION THREAD

**Discussion by cabraham03** [starship/starship#5189] [Q&A]
I'm trying to generate a prompt similar to toml file described in tokyo-night https://starship.rs/presets/tokyo-night.html, the only thing that I want to change, is that ones that the command is executed, don't keep the prompt line, just the executed commands, but the prompt don't, in the example the first prompt line must not appear after of the enter. 
I attached an edited pictures, in the first image the arrow indicate the prompt that I want to avoid, I want something similar to the second picture, ones the command is executed just keep the command and parameters.
Its possible or any way to do it ?? 
thanks  

![Screen Shot 2023-05-17 at 2 01 54 PM](https://github.com/starship/starship/assets/9662585/aff711a3-2285-4786-a2ca-cf5dd9fad5f0)

**Comment by andytom** [comment]
This is possible in some shells and adding support for other shells is being tracked in #888.
